src/injected/analyzers/filter-results.ts

```diff
@@ -10,7 +10,8 @@ export const filterNeedsReviewResults = (results: ScanResults): ScanResults => {
         v =>
             v.id !== 'aria-input-field-name' &&
             v.id !== 'color-contrast' &&
-            v.id !== 'th-has-data-cells',
+            v.id !== 'th-has-data-cells' &&
+            v.id !== 'label-content-name-mismatch',
```

Comment:
Did you intend to include the `filter-results` change (and its tests) here? I suspect they're intended to be for #4339, not this PR

Reply:
Right, this belongs to #4339
I undid changes to this file and `src/tests/unit/tests/scanner/custom-rules/color-rule.test.ts` in my last commit.